### DISCUSSION TITLE
Taskbar Labels for Windows 11 works not corretly

### DISCUSSION THREAD

**Discussion by Smorvel** [ramensoftware/windhawk-mods#98] [Q&A] [answered]
How to make applications not go out of bounds on windows 11?

![image](https://user-images.githubusercontent.com/19754243/217143753-5cc27a43-e475-47c1-a620-9b2003f0b714.png)

**Comment by m417z** [comment]
It's a known limitation of the mod, and there's no solution at this time.

**Comment by m417z** [comment, marked as answer]
Fixed in the latest version of the mod.
